cmd/warm-backend.go

```diff
@@ -78,7 +78,7 @@ func checkWarmBackend(ctx context.Context, w WarmBackend) error {
 			}
 		}
 	}
-
+	r.Close()
```

Comment:
We should always drain the body via, even if its `0-byte` object, it is important to see an EOF - once the reader sees EOF the net/http automatically re-purposes the connection.

```
io.Copy(io.Discard, r)
r.Close()
```